.github/workflows/version.yml

```diff
@@ -39,3 +39,5 @@ jobs:
       - name: 'Alpha Major Release'
         run: npm run alpha:major
         if: contains(github.head_ref, '/major') && github.base_ref == 'next'
+      - name: 'Build Docs':
+        run: npm run build:docs
```

Comment:
This was the missing script for updating the docs automatically on release

Reply:
Nice we god this fixed!